docs/conf.py

```diff
@@ -56,7 +56,7 @@
 
 # General information about the project.
 project = 'librosa-gallery'
-copyright = '2016, librosa development team'
+copyright = '2017, librosa development team'
```

Comment:
please revert the copyright date.  It holds from the start of copyright, not the date of latest revision.